.eslintrc.json

```diff
@@ -3,7 +3,9 @@
         "es6": true,
         "node": true
     },
-    "extends": "eslint:recommended",
+    "extends": [
+      "eslint:recommended"
+    ],
```

Comment:
Converted this to an array syntax so it'd be easier to do a quick "plugin:promise/recommended" if you want to extend that recommended config in the future.